lua/telekasten.lua

```diff
@@ -224,6 +227,10 @@ local function get_uuid(opts)
 end
 
 local function generate_note_filename(uuid, title)
+    if M.Cfg.filename_space_subst ~= nil then
+        title = title:gsub(" ", M.Cfg.filename_space_subst)
```

Comment:
I believe we should escape properly the title first using `escape` function. 
This function ensures everything works properly when we use `gsub`.

I do not have time now to dive deeper and really see why or how, but issues like #179 were simply solved by using this proper escape function first.

Reply:
Change has been made! Just wrapped with `escape(title)` before `gsub`ing.